### ISSUE TITLE
DOSBox Tandy/RGB/CGA mode loading up with "display info" overwriting screen

### ISSUE THREAD

**Issue by biffrapper** [HyperspaceMadness/Mega_Bezel#10]
Apparently this is a bug that has been introduced in one the newer versions.  The screenshot I am sharing is applicable with latest release, as well, but I took from a slightly earlier build.  No matter the shader options I select in the menu, I cannot remove this screen, thus cannot see the command prompt and start games.

Oddly, I do not have this issue when I run in VGA mode.

![pootandy-231225-113102](https://github.com/HyperspaceMadness/Mega_Bezel/assets/85194778/8574dbfb-4297-44f9-8ff2-12324f61b073)

**Comment by HyperspaceMadness** [comment]
Do you get this issue when opening the original preset from the package? Or only when you open a preset you've saved in the past?

If you are loading a preset you have saved before, what is very likely happening is that you have saved a full preset rather than a simple preset, and then the Mega Bezel has been updated.

To successfully use your saved presets between Mega Bezel versions you'll need to use simple presets (which only have a reference to the base Mega Bezel preset originally loaded, and any parameter adjustments you have made) rather than full presets (which have a copy of all the passes inside)

You can learn more about simple presets here:

https://docs.libretro.com/guides/shaders/

**Comment by biffrapper** [comment]
Yep, using simple presets.  Something has fundamentally changed between 1.12 and 1.16.  For example, stuff like overscan needs to be set to negative, doublescan has to be manually toggled off, but there's even more that is simply odd, and I discovered this using a slightly older simple preset in the simple preset chain.  The image output is vastly different.  I have posted a before/after, where the first photo is in 1.12.


![20240321085440_1](https://github.com/HyperspaceMadness/Mega_Bezel/assets/85194778/f4a7b702-43f7-4ed3-b3e0-4f9659cbd9a9)
![20240321091017_1](https://github.com/HyperspaceMadness/Mega_Bezel/assets/85194778/baa42147-b8f1-42ff-bc09-6793f5afd96c)


Since I am emulating a mid 80s RGB CRT, there are going to be particular imperfections and quirks to the tube that later VGA tubes will not have.  The 1.16 implementation seems to be doing pseudo-VGA characteristics, yet none of my parameters had changed--save for manually having to add negative overscan values and removing doublescan.

**Comment by HyperspaceMadness** [comment]
If you could add your preset here I could take a look at what might be different. For the doublescan, some of this may have changed because the Guest shader has been updated

**Comment by biffrapper** [comment]
They are a long simple chain, so I have uploaded the entire chain.  Also, I have uploaded the entire chain saved into one preset as well.

The single preset is in "pootandy.zip," which the dosbox-core references when running my tandy mode .bat file, and the chain is "Tandy Tweaked shader chain.zip"

[pootandy.zip](https://github.com/HyperspaceMadness/Mega_Bezel/files/14762321/pootandy.zip)
[Tandy Tweaked shader chain.zip](https://github.com/HyperspaceMadness/Mega_Bezel/files/14762354/Tandy.Tweaked.shader.chain.zip)

Worth noting is when I run the chain in 1.16, I do not get the persistent screen info on the screen, but instead I get highly amplified moire patterns and tube brightness(think traditional black tubes where you dial up brightness so much grey starts appearing in the black), which is strange, and cannot be removed by simply lowering brightness/gamma/saturation.

I may have solved the problem, mostly, by heavily tweaking the scanline settings and some ghosting persistence elsewhere.  Some of the values needed to be nearly the inverse of what they were, and this can be seen in the new Prototype settings below.  They aren't perfect(too bright), but more a rough tweak WIP to try and get back to the 1.12 appearance.

[Tandy VGA RGA 1.16 PROTOTYPE.zip](https://github.com/HyperspaceMadness/Mega_Bezel/files/14764763/Tandy.VGA.RGA.1.16.PROTOTYPE.zip)


Long story short:  There seem to be changes in 1.16, but I wasn't using simple preset chain as I thought, but instead the "save shader as game preset," file stored in config under the core.
